pages/multiselect/disabled-reason.page.tsx

```diff
@@ -7,6 +7,14 @@ import Multiselect, { MultiselectProps } from '~components/multiselect';
 
 import ScreenshotArea from '../utils/screenshot-area';
 
+const extraOptions = [...Array(30).keys()].map(n => {
+  const numberToDisplay = (n + 5).toString();
```

Comment:
just curious, why `n + 5`?

Reply:
These `extraOptions` get appended in between the other options, in partcular right after the 4 options defined between lines 19 and 34. The purpose is to make the list long so that the fix can be tested with the disabled option at the end. The change in this file was for manual testing though, the added integration test uses the select component.